hack/relprep.sh

```diff
@@ -80,29 +91,32 @@ for i in $crds; do
   fi
 done
 
-echo "Verifying that bundle-generated CSV (for testing) is equivalent to shipping CSV"
-ignorefields='createdAt|operators\.operatorframework\.io/builder|app\.kubernetes\.io/version'
-bcsv=bundle/manifests/keda.clusterserviceversion.yaml
-mcsv=config/manifests/bases/keda.clusterserviceversion.yaml
-if ! diff -u <(grep -vE "$ignorefields" < $bcsv) <(grep -vE "$ignorefields" < $mcsv) | \
-    sed 's#^--- /dev/fd/[0-9]*#--- '"$bcsv"'#;s#^+++ /dev/fd/[0-9]*#+++ '"$mcsv"'#'; then
-  echo "Error: It appears that there are non-trivial differences between $bcsv and $mcsv."
-  echo "As appropriate, make changes to $mcsv or the inputs to $bcsv, and re-run this script"
-  exit 1
-fi
-
 all_mods="$(go list -mod=readonly -m -f '{{ if and (not .Indirect) (not .Main)}}{{.Path}}{{end}}' all)"
 declare -A updated_mods
+to_update=()
+
+# force versions of k8s components
+for i in $kube_components; do
+    to_update+=("k8s.io/$i@$k8sver")
+    updated_mods["k8s.io/$i"]=1
+done
+
+# hack: force version of openshift API module based upon k8s->openshift version skew (e.g. 1.27 -> 4.14)
+openshift_branch="release-4.$(($(echo $k8sver | sed 's/v0\.\([0-9]*\)\.[0-9]*$/\1/')-13))"
+to_update+=("github.com/openshift/api@$openshift_branch")
+updated_mods[github.com/openshift/api]=1
 
-echo
-echo Updating all deps
 for m in $all_mods; do
     [ -n "${updated_mods[$m]+1}" ] && continue # already updated
-    echo go get -u $m
-    go get -u $m
+    to_update+=("$m" )
     updated_mods["$m"]=1
 done
 
+echo
+echo Updating all deps
+echo go get "${to_update[@]}"
+go get "${to_update[@]}"
```

Comment:
I'm not the police, but did you mean to take out the `-u` here?

Reply:
I did not mean to do that. Thank you for catching it!